modules/update-using-custom-machine-config-pools-about.adoc

```diff
@@ -6,28 +6,28 @@
 [id="update-using-custom-machine-config-pools-about_{context}"]
 = About performing a canary rollout update
 
-This topic describes the general workflow of this canary rollout update process. The steps to perform each task in the workflow are described in the following sections.
+This following steps outline the high-level workflow of the canary rollout update process:
 
-. Create MCPs based on the worker pool. The number of nodes in each MCP depends on a few factors, such as your maintenance window duration for each MCP, and the amount of reserve capacity, meaning extra worker nodes, available in your cluster.
+. Create custom machine config pools (MCP) based on the worker pool.
 +
 [NOTE]
 ====
-You can change the `maxUnavailable` setting in an MCP to specify the percentage or the number of machines that can be updating at any given time. The default is 1.
+You can change the `maxUnavailable` setting in an MCP to specify the percentage or the number of machines that can be updating at any given time. The default is `1`.
 ====
 
 . Add a node selector to the custom MCPs. For each node that you do not want to update simultaneously with the rest of the cluster, add a matching label to the nodes. This label associates the node to the MCP.
 +
-[NOTE]
+[IMPORTANT]
 ====
-Do not remove the default worker label from the nodes. The nodes *must* have a role label to function properly in the cluster.
+Do not remove the default worker label from the nodes. The nodes must have a role label to function properly in the cluster.
 ====
 
 . Pause the MCPs you do not want to update as part of the update process.
 
 . Perform the cluster update. The update process updates the MCPs that are not paused, including the control plane nodes.
 
-. Test the applications on the updated nodes to ensure they are working as expected.
+. Test your applications on the updated nodes to ensure they are working as expected.
 
-. Unpause the remaining MCPs one-by-one and test the applications on those nodes until all worker nodes are updated. Unpausing an MCP starts the update process for the nodes associated with that MCP. You can check the progress of the update from the web console by clicking *Administration* -> *Cluster settings*. Or, use the `oc get machineconfigpools` CLI command.
+. Unpause the remaining MCPs one at a time and test the applications on those nodes until all worker nodes are updated. Unpausing an MCP starts the update process for the nodes associated with that MCP.
 
 . Optionally, remove the custom label from updated nodes and delete the custom MCPs.
```

Comment:
If this is optional, should [this section](https://github.com/openshift/openshift-docs/pull/71004/files#diff-cc50790f38542369fded8e90bdb8dbbd70fe0e0ff6518577f270d53c06f7c245) be described as optional as well?

Reply:
From Kathryn: 

Kathryn Alexander
:books:  [15 days ago](https://redhat-internal.slack.com/archives/C06J4CEB4EN/p1707942863354129?thread_ts=1707942460.929699&cid=C06J4CEB4EN)
For titles, I think we decided that we shouldn't use it. If modules are meant to be reusable, you can't always say that it's going to be optional.

There might be a PR open for our style guide that suggests against. I think @michaelryanpeter was going to handle that? Not sure if he has yet.